.github/workflows/python-CI.yml

```diff
@@ -149,16 +149,29 @@ jobs:
     if: ${{ needs.changes.outputs.phoenix == 'true' }}
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest, windows-2019, macos-12]
+        os: [ ubuntu-latest, windows-latest, windows-2019, macos-12 ]
+    services:
+      postgres:
+        # Applying this workaround: https://github.com/actions/runner/issues/822
+        image: ${{ (matrix.os == 'ubuntu-latest') && 'postgres:12' || '' }}
+        env:
+          POSTGRES_PASSWORD: phoenix
```

Comment:
setting a distinct password to avoid making inadvertent connections when running locally